### ISSUE TITLE
Change the Enter key to Return key on MacOS

### ISSUE THREAD

**Issue by minhtriet** [spyder-ide/spyder#7253]
### Issue Report Checklist

* [x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [x] Read the relevant sections of the [Spyder Troubleshooting Guide](https://github.com/spyder-ide/spyder/wiki/Troubleshooting-Guide-and-FAQ) and followed its advice
* [ ] Reproduced the issue after updating with ``conda update spyder`` (or ``pip``, if not using Anaconda)
* [ ] Could not reproduce inside ``jupyter qtconsole`` (if console-related)
* [ ] Tried basic troubleshooting (if a bug/error)
    * [ ] Restarted Spyder
    * [ ] Reset preferences with ``spyder --reset``
    * [ ] Reinstalled the latest version of [Anaconda](https://www.anaconda.com/download/)
    * [ ] Tried the other applicable steps from the Troubleshooting Guide
* [x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
I would like to change the Enter key to Return key on MacOS. One less key to press. If it fits with the UX and is relatively straightforward, I would gladly make a PR.

## Version
Spyder 3.2.8 
Python 3.6.3 64bits, Qt 5.9.4, PyQt5 5.9.2 on Darwin 

### Dependencies

```
IPython >=4.0    :  6.3.1 (OK)
cython >=0.21    :  None (NOK)
jedi >=0.9.0     :  0.12.0 (OK)
nbconvert >=4.0  :  5.3.1 (OK)
numpy >=1.7      :  1.14.2 (OK)
pandas >=0.13.1  :  None (OK)
psutil >=0.3     :  5.4.5 (OK)
pycodestyle >=2.3:  2.4.0 (OK)
pyflakes >=0.6.0 :  1.6.0 (OK)
pygments >=2.0   :  2.2.0 (OK)
pylint >=0.25    :  1.8.4 (OK)
qtconsole >=4.2.0:  4.3.1 (OK)
rope >=0.9.4     :  0.9.4-1 (OK)
sphinx >=0.6.6   :  1.7.4 (OK)
sympy >=0.7.3    :  None (NOK)```

**Comment by ccordoba12** [comment]
> One less key to press

What do you mean by this?

**Comment by minhtriet** [comment]
I mean pressing Control Return instead of Control Shift Return

**Comment by ccordoba12** [comment]
To evaluate cells? Please be specific.

**Comment by minhtriet** [comment]
Sorry for the vagueness. Yes, Shift Return somehow works, but also advance to the next cell. I would like to evaluate a cell by pressing Control-Return rather than Control-Shift-Return.

**Comment by CAM-Gerlach** [comment]
>  Yes, Shift Return somehow works, but also advances to the next cell.

If you look in our keyboard shortcuts editor (``Preferences > Keyboard shortcuts``), or the shortcuts summary (``Help > Shortcuts Summary``), you'll see that's the designated shortcut for ``run cell and advance``, i.e. run the current cell and then go to the next one, which is exactly what you are describing, so it is behaving exactly as expected.

> I would like to evaluate a cell by pressing Control-Return rather than Control-Shift-Return.

I believe the former (i.e. ``Cmd-Return`` on a Mac) should be the default shortcut, but did you check it in the keyboard shortcuts editor in the preferences? ``Enter`` on a PC should be equivalent to ``Return`` on a Mac, though the naming might not be properly responsive which is really a part of #7146 which should be in the process of being addressed for 3.3.1.

**Comment by minhtriet** [comment]
Thank you @CAM-Gerlach for the answer. Preferences > Keyboard shortcuts shows that run current cell is ^⏎. The referenced issue looks correct and very detailed.
